### ISSUE TITLE
Requested 4294967295x1 (32.0GB) array exceeds maximum array size

### ISSUE THREAD

**Issue by hasinadrone** [Georacer/ardupilog#93]
Hello everyone,

I'm facing an issue with MATLAB: when I run my program, I get the following error: "Requested 4294967295x1 (32.0GB) array exceeds maximum array size preference (7.9GB)". It seems that the log file I'm trying to process is too large for my machine's capacity, which has 8 GB of memory.

I could potentially bypass this problem by using a PC with 32 GB of memory, but I'm wondering if there's an alternative solution to process this large log file on my current 8 GB memory PC.

**Comment by Georacer** [comment]
Hello! Thanks for the report!

How big is your log file itself? I could try to reproduce the issue and potentially solve it, if I knew what is the target file size.

**Comment by Georacer** [comment]
Of course, ideally if you could upload the log file somewhere so that we can make sure it can be read after the patches, it would be the best.

**Comment by Georacer** [comment]
Closing as stale. Feel free to comment again if you still have this issue.
